### PR TITLE
To do list structure

### PR DESCRIPTION
**Pull request by Hidar37**
- Everything is add using **_Webpack_** including [ HTML, CSS, JS] files.
- **Linter** error removed from the code.
- **ES6 arrow function** concept use for defining `showtoDoList()` which show all the data that store in array.
- Style add to project completely.
- **Remove** button functionality add completely.
- Readme file updated for now.
**_NOTE:_**
In order to run the program please run the **webpack serve** because I am using it for **updating** the **webpack** files. if you run the program with out **webpack serve** you will not see the correct output.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,32 @@
-# To-Do-List
\ No newline at end of file
+# To Do List Project
+
+> This is simple to do list project which is under building. For now there is no functionlity to use.
+
+
+## Built With
```
Comment (ichala) [inline README.md:6]: - [ ] your readme file should have a section that guides your visitors on how to Setup&Install the project, kindly add the necessary instructions to make your readme more professional 🙏

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    const removeBtn = document.createElement('i');
+    removeBtn.setAttribute('class', 'change-position');
+    removeBtn.classList.add('fa-solid', 'fa-ellipsis-vertical');
+    // Adding textNode to Elements
+    pElement.appendChild(document.createTextNode(`${item.description}`));
+    // remove Button
+    removeBtn.addEventListener('click', () => {
+      toDoList.splice(position, 1);
+      liElement.remove();
+      // window.location.reload();
```
[the remark this answers is not in the document]
Comment (Hidar37) [reply src/index.js:39]: sorry i comment that line for testing purpose or else i need that part. But thanks a lot for your clear instruction.

[the remark this answers is not in the document]
Comment (ichala) [reply src/index.js:39]: @Hidar37 don't worry , your code is already awesome once you do the changes it will be perfect 👏

**Review by Alfredbis29 — APPROVED** [review]
# STATUS: APPROVED  
 ## Hi @Hidar37 ,
Your project is complete! There is nothing else to say other than... Great job clap Congratulations! :tada:
 
 TO HIGHLIGHT:
 
 * Your project is professional :white_check_mark:
 * Your repo is professional :white_check_mark:
 * Well structured files :white_check_mark:
 * No Linter errors :white_check_mark:
 
 Now you can merge this branch and move on :1st_place_medal: HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers four_leaf_clover:


 
 ## Optional suggestions

-N/A

 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
 
Cheers and Happy coding! :clap:  :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**